src/helm/proxy/retry.py

```diff
@@ -30,20 +37,58 @@ def wait(attempts: int, delay: float) -> float:
         Wait function to pass into `Retrying` that logs and returns the amount of time to sleep
         depending on the number of attempts and delay (in milliseconds).
         """
-        hlog(f"{operation} failed. Retrying (attempt #{attempts + 1}) in {delay // 1000} seconds...")
+        hlog(
+            f"{operation} failed. Retrying (attempt #{attempts + 1}) in {delay // 1000} seconds... "
+            "(See above for details)"
+        )
         return _retrying.exponential_sleep(attempts, delay)
 
+    def stops_on_keyboard_interrupt(exception: Exception) -> bool:
```

Comment:
Don't make this a separate function; just inline this line of code and comment where it is used.